SpatialStructure-7/SpatialStructure-7.ifc

```diff
@@ -0,0 +1,2261 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [Ifc4X3NotAssigned]'),'2;1');
+FILE_NAME(
+/* name */ 'SpatialStructure-7.ifc',
+/* time_stamp */ '2021-04-16T13:44:21',
+/* author */ ('Sebastian Esser'),
+/* organization */ ('TUM-CMS'),
+/* preprocessor_version */ 'GeometryGymIFC v0.1.8.0 by Geometry Gym Pty Ltd built 2021-03-05T10:03:04',
+/* originating_system */ 'GeometryGymIFC v0.1.8.0',
+/* authorization */ 'None');
+
+FILE_SCHEMA (('IFC4X3_RC3'));
+ENDSEC;
+
+DATA;
+#1= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#2= IFCAXIS2PLACEMENT3D(#1,$,$);
+#3= IFCLOCALPLACEMENT($,#2);
+#4= IFCSITE('3BIjkEK2nD9gX0CmwCIbfT',#10,'HNTB_bridge',$,$,#3,$,$,$,$,$,$,$,$);
+#5= IFCPERSON('SEsser','Esser','Sebastian',$,$,$,$,$);
+#6= IFCORGANIZATION($,'TUM-CMS',$,$,$);
+#7= IFCPERSONANDORGANIZATION(#5,#6,$);
+#8= IFCAPPLICATION(#9,'0.1','IfcInfraToolkit_Dyn','IfcInfraToolkit_Dyn');
+#9= IFCORGANIZATION($,'Geometry Gym Pty Ltd',$,$,$);
+#10= IFCOWNERHISTORY(#7,#8,$,.ADDED.,1618573434,$,$,1618573434);
+#11= IFCPROJECT('11RsnJDY1B4xptj2Y3$FDR',#10,'HNTB_conreteBridgeStoryLine',$,$,$,$,(#71),#12);
+#12= IFCUNITASSIGNMENT((#13,#14,#15,#16,#17));
+#13= IFCSIUNIT(*,.LENGTHUNIT.,.MILLI.,.METRE.);
+#14= IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#15= IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#16= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#17= IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
+#18= IFCRELAGGREGATES('0hxh30CXDBAuopRmv8Sglt',#10,$,$,#11,(#4));
+#19= IFCBRIDGE('1Vp_UdHhH1D9MkYNdnmsXD',#10,'HNTB bridge',$,$,#3,$,$,$,$);
+#20= IFCRELAGGREGATES('1ezg_czsb7hPW4rHI$5bKg',#10,$,$,#4,(#19,#42));
+#21= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#22= IFCAXIS2PLACEMENT3D(#21,$,$);
+#23= IFCLOCALPLACEMENT(#3,#22);
+#24= IFCFACILITYPART('1wIokej6rA3wPC2_78VBrg',#10,'HNTB_Superstructure',$,$,#23,$,$,$,IFCBRIDGEPARTTYPEENUM(.SUPERSTRUCTURE.),.VERTICAL.);
+#25= IFCRELAGGREGATES('2VEYNB7Ab9FgC3TVK3Rl62',#10,$,$,#19,(#24,#29,#33,#37,#41));
+#26= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#27= IFCAXIS2PLACEMENT3D(#26,$,$);
+#28= IFCLOCALPLACEMENT(#3,#27);
+#29= IFCFACILITYPART('13S1ixLwz9Svu9IiDazn75',#10,'HNTB_Substructure',$,$,#28,$,$,$,IFCBRIDGEPARTTYPEENUM(.SUBSTRUCTURE.),.VERTICAL.);
+#30= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#31= IFCAXIS2PLACEMENT3D(#30,$,$);
+#32= IFCLOCALPLACEMENT(#3,#31);
+#33= IFCFACILITYPART('2REWep1_P2zQIQ41yBioh9',#10,'HNTB_Pier',$,$,#32,$,$,$,IFCBRIDGEPARTTYPEENUM(.PIER.),.VERTICAL.);
+#34= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#35= IFCAXIS2PLACEMENT3D(#34,$,$);
+#36= IFCLOCALPLACEMENT(#3,#35);
+#37= IFCFACILITYPART('1RYbThjKT14OISoCevj7NE',#10,'HNTB_Foundation',$,$,#36,$,$,$,IFCBRIDGEPARTTYPEENUM(.FOUNDATION.),.VERTICAL.);
+#38= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#39= IFCAXIS2PLACEMENT3D(#38,$,$);
+#40= IFCLOCALPLACEMENT(#3,#39);
+#41= IFCFACILITYPART('3Mt7zpr81DtxyiuLfu$1cu',#10,'HNTB_Abutment',$,$,#40,$,$,$,IFCBRIDGEPARTTYPEENUM(.ABUTMENT.),.VERTICAL.);
+#42= IFCROAD('3kf3anmv916eh69wbM4NJB',#10,'Road on top of the bridge',$,$,#3,$,$,$,$);
```

Comment:
Is It worth adding a `IfcRelInterferesElements` as Road 'passesOver' Bridge for completeness?

Reply:
see 
c105e03